srcts/extras/busy-indicators/busy-indicators.scss

```diff
@@ -90,16 +90,25 @@
   page is busy, but there are no recalculating elements.
 */
 [data-shiny-busy-spinners][data-shiny-busy-pulse] {
-  &.shiny-busy:not(:has(.recalculating))::after {
+  &.shiny-busy::after {
     @include shiny-page-busy;
   }
+  &.shiny-busy:has(.recalculating)::after {
+    display: none;
+  }
+  &.shiny-busy:has(#shiny-disconnected-overlay)::after {
```

Comment:
Someday might be worth adding a class or attribute somewhere (`<html>`? `<body>`?) for disconnected status.